charts/prometheus-kafka-exporter/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+appVersion: "v1.2.0"
+description: A Helm chart to export the metrics from Kafka in Prometheus format.
+name: prometheus-kafka-exporter
+home: https://gkarthiks.github.io/helm-charts/charts/prometheus-kafka-exporter
+version: 0.1.4
```

Comment:
Why does this chart start with `version: 0.1.4`? Is it donated from somewhere else (if so, where?). If not, we should start with `0.1.0`.

Reply:
yes, its from https://artifacthub.io/packages/helm/gkarthiks/prometheus-kafka-exporter

Reply:
Awesome 👍

Do you mind noting in the description this is from https://github.com/danielqsj/kafka_exporter?

Will you also link a PR there to deprecate that chart (similar to https://github.com/helm/charts/blob/master/PROCESSES.md)?